src/routes/trackpad.tsx

```
@@ -1,98 +1,89 @@
-            {/* Touch Surface */}
-            <TouchArea
-                isTracking={isTracking}
-                scrollMode={scrollMode}
-                handlers={handlers}
-                status={status}
-            />
+	return (
+		<div
+			className="flex flex-col h-full overflow-hidden"
+			onClick={handleContainerClick}
+		>
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)